Node.js/GettingStarted/azure_batch_client.js

```diff
@@ -0,0 +1,196 @@
+// Initializing node.js modules
```

Comment:
The sample is explained in detail line by line in the azure docs article I have submitted for review. I will add a readme with a reference to that once it is published.